openshift/templates/services/audio/capture/deploy.yaml

```diff
@@ -0,0 +1,338 @@
+  - name: KAFKA_CLIENT_ID
+    displayName: Kafka Client ID
+    description: The Kafka client id to uniquely identify this producer.
+    required: true
+    value: audio-capture
+  - name: MAX_FAILED_ATTEMPTS
+    displayName: Maximum Failed Attempts
+    description: The maximum number of failed attempts before stopping the service.
+    required: true
+    value: "5"
+  - name: DATA_SOURCE_MEDIA_TYPE
+    displayName: Data Source Media Type
+    description: The name of the media type that this service will ingest.
+    required: true
+    value: News Radio
+
+  - name: DB_SECRETS_NAME
+    displayName: Database Secret Name
+    description: Name of the database secrets object.
+    required: true
+    value: tno-database
+  - name: API_CONFIG_NAME
+    displayName: API Config Name
+    description: Name of the api config map object.
+    required: true
+    value: tno-api-editor
+
+  - name: VOLUME_SIZE
+    displayName: Backup Volume Size
+    description:
+      The size of the persistent volume used for restoring and verifying
+      backups, e.g. 512Mi, 1Gi, 2Gi.  Ensure this is sized correctly.  It should be
+      large enough to contain your largest database.
+    required: true
+    value: "5Gi"
+  - name: VOLUME_CLASS
+    displayName: Backup Volume Class
+    description:
+      The class of the persistent volume used for restoring and verifying
+      backups; should be one of netapp-block-standard or netapp-file-standard.  netapp-block-standard
+      performs better.
+    required: true
+    value: "netapp-file-standard"
+  - name: VOLUME_MOUNT_PATH
+    displayName: Verification Volume Mount Path
+    description: The path on which to mount the volume.
+    required: true
+    value: "/data"
+
+  - name: REPLICAS
+    displayName: Pod Replicas
+    description: Number of pod replicas to run
+    value: "1"
+  - name: CPU_REQUEST
+    displayName: Requested Minimum Resources CPU Limit
+    description: The requested minimum resources CPU limit (in cores) for this build.
+    required: true
+    value: 20m
+  - name: CPU_LIMIT
+    displayName: Resources CPU Limit
+    description: The resources CPU limit (in cores) for this build.
+    required: true
+    value: 100m
+  - name: MEMORY_REQUEST
+    displayName: Requested Minimum Memory Limit
+    description: Minimum requested amount of memory the container can use.
+    required: true
+    value: 250Mi
+  - name: MEMORY_LIMIT
+    displayName: Memory Limit
+    description: Maximum amount of memory the container can use.
+    required: true
+    value: 500Mi
+
+objects:
+  # Storage for videos.
+  - kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: ${SOLUTION_NAME}-${APP_NAME}
+      namespace: ${PROJECT_NAMESPACE}-${ENV_NAME}
+      labels:
+        name: ${SOLUTION_NAME}-${APP_NAME}
+        solution: ${SOLUTION_NAME}
+        app: ${APP_NAME}
+    spec:
+      storageClassName: ${VOLUME_CLASS}
+      accessModes:
+        - ReadWriteOnce
```

Comment:
Not certain if other pods will be able to connect to this PVC.  Will test once we have a working service.